### ISSUE TITLE
Modal: Missing focus trap

### ISSUE THREAD

**Issue by omaxel** [zetareticoli/cssui#10]
When the user clicks on a button that triggers the modal opening, he's still able to move the focus around the page using the tab key.

Instead, the focus should be trapped inside the modal and its controls.

**Comment by zetareticoli** [comment]
Closed issue in 69b5b7c
